portfolio/src/main/java/com/google/sps/servlets/Country.java

```diff
@@ -0,0 +1,11 @@
+package com.google.sps.servlets;
+
+class Country {
+    private String country;
+    private String capital;
+
+    public Country(String country, String capital) {
+        this.country = country;
```

Comment:
Naming suggestion: a field called `country` on a class called `Country` can be confusing. Maybe `name` or `countryName` would be clearer?